packages/vue-labs/src/components/FTable/ITableHeader.vue

```diff
@@ -79,5 +79,8 @@ function onKeydownCell(e: KeyboardEvent): void {
                 <f-icon :name="sortIcon" :class="sortIconClass"></f-icon>
             </i-flex-item>
         </i-flex>
+        <i-flex gap="1x" class="table-ng__column__description">
```

Comment:
Det känns lite onödigt att ha en flexbox med bara ett item, är avsikten att de ska in fler items här senare?

Oavsett tycker jag vi ska köra en helt vanlig span/div och ersätta med styling `display: flex; gap: 1rem` om vi har behov av flexbox.

EDIT: jag ser nu att vi skriver över flexbox med `display: block` så då blir det definitivt onödigt. Föreslår att byta ut mot en div:

```suggestion
        <div class="table-ng__column__description">
```

Då behöver du inte `display: block` i stylingen heller eftersom det är default.